packages/trader/src/Stores/Modules/Trading/Actions/start-date.ts

```diff
@@ -33,16 +33,14 @@ export const onChangeStartDate: TOnChangeStartDate = async store => {
     const obj_sessions = ContractType.getSessions(contract_type, start_date);
     const sessions = obj_sessions.sessions;
     const obj_start_time = ContractType.getStartTime(sessions, start_date, start_time);
-    start_time = obj_start_time.start_time;
 
     const obj_duration_units_list = ContractType.getDurationUnitsList(contract_type, contract_start_type);
     const duration_units_list = obj_duration_units_list.duration_units_list;
     const obj_duration_unit = ContractType.getDurationUnit(duration_unit, contract_type, contract_start_type);
 
     const obj_expiry_type = ContractType.getExpiryType(duration_units_list, expiry_type);
     expiry_type = obj_expiry_type.expiry_type;
-    const obj_expiry_date = ContractType.getExpiryDate(duration_units_list, expiry_date, expiry_type, start_date);
-    expiry_date = obj_expiry_date.expiry_date;
+    const obj_expiry_date = ContractType.getExpiryDate(duration_units_list, store.expiry_date, expiry_type, start_date);
```

Comment:
Changes in this file do not change implementation?

Reply:
no, they don't)